index.html

```diff
@@ -50,6 +50,146 @@ <h3 class="helpingtext">LET'S CONNECT</h3>
     </div>
     </div> 
-</section>     
+</section>
+
+<section class="Grid-boxes">
+    <article class="Box">
+        <img class="box-image" src="req/tonic.png" alt="tonic">
+        <div class="content">
+            <h2 class="content-title">Tonic</h2>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required</p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>
+    <article class="Box">
+        <img class="box-image" src="req/multi.png" alt="multi">
+        <div class="content">
+            <h2 class="content-title">Multi-Post Stories</h2>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required</p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>
+    <article class="Box">
+        <img class="box-image" src="req/tonic_2.png" alt="tonic_2">
+        <div class="content">
+            <h2 class="content-title">Tonic</h2>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required</p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>
+    <article class="Box">
+        <img class="box-image" src="req/multi_2.png" alt="multi_2">
```

Comment:
- [Optional] COnsider using an indentation of two spaces in all your documents, this is a convention and also best practice.